lib/SPIRV/libSPIRV/SPIRVInstruction.cpp

```diff
@@ -245,6 +245,31 @@ SPIRVInstruction *createInstFromSpecConstantOp(SPIRVSpecConstantOp *Inst) {
          "Op code not allowed for OpSpecConstantOp");
   Ops.erase(Ops.begin(), Ops.begin() + 1);
   switch (OC) {
+  case OpVectorShuffle: {
```

Comment:
As part of #1072 these classes should be changed to inherit from `SPIRVInstTemplateBase`.